gnovm/pkg/gnolang/realm.go

```diff
@@ -84,8 +84,29 @@ var (
 	// fixed memory caps. For now though it isn't a problem:
 	// https://github.com/gnolang/gno/pull/3424#issuecomment-2564571785
 	pkgIDFromPkgPathCache = make(map[string]*PkgID, 100)
+	// Cache for Private Status
+	pkgPrivateStatusCache = make(map[string]bool, 100)
```

Comment:
would this continue to work if values are randomly evicted from `pkgPrivateStatusCache`? If the answer if no, it shouldn't be called a "cache". (looks like answer is no)